lib/actions/call-taker.js

```diff
@@ -133,6 +141,50 @@ export function fetchCalls () {
   }
 }
 
+/**
+ * Fetch all field trip requests (as summaries).
+ */
+export function fetchFieldTrips () {
+  return function (dispatch, getState) {
+    dispatch(requestingFieldTrips())
+    const {callTaker, otp} = getState()
+    if (sessionIsInvalid(callTaker.session)) return
+    const {datastoreUrl} = otp.config
+    const {sessionId} = callTaker.session
+    fetch(`${datastoreUrl}/fieldtrip/getRequestsSummary?${qs.stringify({sessionId})}`)
+      .then(res => res.json())
+      .then(fieldTrips => {
+        console.log('GET field trips response', fieldTrips)
```

Comment:
Remove console.log

Reply:
Fixed in https://github.com/opentripplanner/otp-react-redux/pull/230/commits/9a56caeb148eeffc0cbed3c8e205f68f01084b45